### ISSUE TITLE
Party/NPC Campaign Sheet

### ISSUE THREAD

**Issue by Butterfly-Dragon** [Orcpub/orcpub#118]
**Is your feature request related to a problem? Please describe.**
from the party menu the ability to print down a summary of the whole party as a "single" charsheet.

**Describe the solution you'd like**
i've attached the standard WotC Party Tracker. In notes i usually write down stats, trained skills, race/class features which have uses and what kind of rest they require.

Though it would help to have more "notes" space 😅 

example given:
16,16,14,8,10,10; animal handling, intimidation, perception, survival; ❏❏ rage;
10,16,10,10,12,16; acrobatics, arcana, intimidation, perception; spells: 1st❏❏❏ 2nd❏❏

**Describe alternatives you've considered**

3rd party charsheets.

They were messier, since instead of having an all-purpose "notes" place they tended to have detailed combat stats.

Also they usually considered "personality" "ideals" "bonds" and "flaws" as something unimportant to be left out.

I... honestly don't really need a "party combat sheet" i need a party sheet for planning and doing the actual adventure. Knowing at a glance what the adventurers' motives are and what capabilities they have left and if i should follow with a short/long rest or if they can handle an incoming challenge is often more important for on-the-fly decisions about the *actual adventure* than knowing stuff that starts and ends with combat.

Players can handle the actual combat mechanics once they get there, and in D&D when combat ends most of the stuff ends too.

I need to know what combat to give them and what moral/skill/combat challenges to set them onto, before they get to it, a party sheets that reminds me at a glance what the characters less used traits are and if the group needs a short or long rest... that would be better.

**Additional context**

[DnD 5e - Tools - Character Sheet - Party Tracker.pdf](https://github.com/Orcpub/orcpub/files/2988443/DnD.5e.-.Tools.-.Character.Sheet.-.Party.Tracker.pdf)

**Comment by datdamnzotz** [comment]
Love the idea!  A DM's cheat sheet for the party.

**Comment by DivertedCircle** [comment]
If you're using a computer during the game then you could duplicate this with the Encounter Builder. To do so you'd need to duplicate your players and load them into the encounter builder. Then any notes you want to make can just be made by editing the character and it would be reflected in the Encounter Builder.
![image](https://user-images.githubusercontent.com/26470570/54700032-c7bde800-4af7-11e9-8297-dc67be6557f9.png)

**Comment by Butterfly-Dragon** [comment]
I do not need something for the fights. I need something to use for the
campaign building and improvvisation process.

Something that allows me to check:
★ what are the motives and impulses less explored of the characters
★ and how close they are to needing any kind of rest.

I do not need to know their health values or which spells are prepared or
whatever. Usually i either remember or can ask them. Or look them up.

But for 99% of the campaign building process, what i need are:
★ Stat values,
★ trained Skills,
★ and number of ability uses left.

The rest might as well not exist, since it's very situational and i will
check it if i want a very specific ability to shine.

**Comment by DivertedCircle** [comment]
Understandable, and I'm sure there are options out there for just that if you want a PDF copy. Everything you listed is already duplicated in DMV/Orcpub via the Encounter Builder, albeit with extra information.

**Comment by Butterfly-Dragon** [comment]
Yes... i can do this manually. Which it's what i've been doing until today.

The whole point is to see it in 1 sheet/page, not around 35 with hours of scrolling like it would happen if i used the orcpub combat tracker.

I'm trying to get a machine to do it for me after a bunch of nice programmers tell said machine how to do it for me. Thank you.
